docs/en/sql-reference/functions/other-functions.md

```diff
@@ -386,13 +386,37 @@ Code: 44. DB::Exception: Received from localhost:9000. DB::Exception: Illegal ty
 
 ## ignore
 
-Accepts any arguments, including `NULL` and does nothing. Always returns 0.
-The argument is internally still evaluated. Useful e.g. for benchmarks.
+Accepts any arguments, including `NULL` and always returns `0`.
```

Comment:
```md
Accepts arbitrary arguments and unconditionally returns `0`.
```

(--> "including NULL" is slightly distracting here. It will be mentioned in below "Arguments" section. We can omit its mention from here.)